ci/jenkins/test.sh

```diff
@@ -565,23 +565,21 @@ function deliver_antrea_linux_containerd {
     echo "==== Start building and delivering Linux containerd images ===="
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
     docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
-    # containerd is the runtime, need to load the image via ctr.
-    ctr -n=k8s.io images import antrea-ubuntu.tar
     echo "===== Pull necessary images on Control-Plane node ====="
     harbor_images=("agnhost:2.13" "nginx:1.15-alpine")
     antrea_images=("e2eteam/agnhost:2.13" "docker.io/library/nginx:1.15-alpine")
     common_images=("registry.k8s.io/e2e-test-images/agnhost:2.29")
     k8s_images=("registry.k8s.io/e2e-test-images/agnhost:2.45" "registry.k8s.io/e2e-test-images/jessie-dnsutils:1.5" "registry.k8s.io/e2e-test-images/nginx:1.14-2")
     e2e_images=("k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.45" "k8sprow.azurecr.io/kubernetes-e2e-test-images/jessie-dnsutils:1.5" "k8sprow.azurecr.io/kubernetes-e2e-test-images/nginx:1.14-2")
 
-    for i in "${!harbor_images[@]}"; do
-        ctr -n=k8s.io images delete "${antrea_images[i]}"
-        ctr -n=k8s.io images pull "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}"
-        ctr -n=k8s.io images tag "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}" "${antrea_images[i]}"
-    done
+    echo "===== Deliver Antrea YAML to Controller nodes ====="
```

Comment:
I suppose we only need to transfer the file to controller Node when it's non-jumper mode?

Reply:
One more question, I am wondering why this step is not in the function 'deliver_antrea'?

Reply:
deliver_antrea is for linux test, deliver_antrea_linux_containerd is a parallel function used in windows test to accelerate image delivery.

Reply:
> I suppose we only need to transfer the file to controller Node when it's jumper mode?

We copy the YAML file to the control-plane node only if the jumper node is not the control-plane node.